### PR TITLE
Update license to 2020-2021 and pass license checks

### PR DESCRIPTION
**Pull request by daniellimws**

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Minimal makefile for Sphinx documentation
 #
+# Copyright (C) 2020-2021  The SymbiFlow Authors.
```
Comment (mithro) [inline docs/Makefile:3]: Copyright should be the first thing in the file.

**Review by mithro — APPROVED** [review]
Just one comment and then we can merge.

**Comment by mithro** [comment]
@daniellimws - I did some changes to the pull request to make the Licensing checks pass. Assuming that Travis / ReadTheDocs pass we can merge it.

File: .github/workflows/licensing.yml [hunk 2]
```diff
@@ -20,5 +28,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+      with:
+        submodules: recursive
```
Comment (daniellimws) [inline .github/workflows/licensing.yml:32]: This is the fix, right?

Comment (mithro) [reply .github/workflows/licensing.yml:32]: Yeap!

**Pull request merged**